client/app/bundles/shared/components/Heading.jsx

```diff
@@ -0,0 +1,23 @@
+// @flow
+import React from 'react';
+import css from './Heading.scss';
+
+type Props = {
+  large?: boolean,
+  small?: boolean,
+  text?: boolean,
+  label?: string,
+};
+
+export default class Heading extends React.Component<Props> {
+  render() {
+    const { large, small, text, label } = this.props;
+    const labelClassNames = `${css.label} ${large ? css.large : ''} ${small ? css.small : ''} ${text ? css.text : ''}`;
+
+    return (
+      <span>
+        <div className={labelClassNames}>{label}</div>
+      </span>
```

Comment:
Instead of having two layers of elements, let's just keep it at one. I think a heading element would make sense here!

Reply:
Which headings should I use. I am thinking
h1 - for large
h2 - for normal
h3- for small
h4- for text

Reply:
I personally think we should be using divs that way if we change the hierarchy of headings we don't need to refactor all of H level headings. I think a div without a parent span should be enough!

Reply:
okay ! Cool !